hardhat.config.ts

```diff
@@ -0,0 +1,56 @@
+// hardhat.config.ts
+import { HardhatUserConfig, SolcConfig } from 'hardhat/types'
+import { hardhat } from './src'
+
+import '@nomiclabs/hardhat-waffle'
+import '@typechain/hardhat'
+
+const DEFAULT_VERSION = 'v0.6'
+
+const optimizer = {
+  runs: 200,
```

Comment:
Do we not optimize a crazy high amount like `10000000` or something?

Reply:
The LinkToken v4 was at 200, so I continued with 200 to get the same compiler output as the contract is already deployed. v6 is in a similar situation but going forward we could raise the number for v7 for example.

Reply:
I've added this at ["Use 1_000_000 optimizer runs for v0.7"](https://github.com/smartcontractkit/LinkToken/pull/29/commits/8c90ff9dab891e4cd79937790f795eefb0f58b84). Left 200 for older versions.